src/framework/components/sound/system.js

```diff
@@ -148,7 +149,9 @@ Object.defineProperty(SoundComponentSystem.prototype, 'volume', {
 Object.defineProperty(SoundComponentSystem.prototype, 'context', {
     get: function () {
         if (!hasAudioContext()) {
+            // #ifdef DEBUG
```

Comment:
Wondering if we should leave that on always to know what's happening on devices that don't support this (where we usually don't use the debug engine)

Reply:
Web Audio is supported absolutely everywhere except IE11. Personally, I don't think the engine should ever `console.log/warn/error` in a release build _except_ for the engine startup message.